content/en/Platform Deep Dive/Scans/blackout-period.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Blackout Period"
+linkTitle: "Blackout Period"
+weight: 45
+description: Schedule blackout periods to automatically pause and resume your DAST scans
+---
+
+Blackout periods allow you to temporarily pause your DAST scans to avoid conflicts with other activities or to prevent disruptions during sensitive times.
+
+- **Automatic Resumption:** Once the blackout period ends, your DAST scans will automatically resume without any manual intervention.
+- **Flexible Scheduling:** Define blackout periods with precise start and end times to fit your specific needs.
+- **No Data Loss:** Any scan progress made before the blackout period will be preserved, and the scan will continue from where it left off.
+
+## Use Cases
+
+- **Planned Maintenance:** Suspend scans during scheduled maintenance windows to prevent interference with system updates or infrastructure changes.
+- **High-Traffic Events:** Pause scans during peak traffic periods or marketing campaigns to avoid performance impacts on your application.
+- **Vulnerability Remediation:** Temporarily disable scans while you're actively fixing identified vulnerabilities to prevent redundant alerts.
+- **Third-Party Integrations:** Prevent scans from interfering with scheduled tasks from other tools or services integrated with your application.
+
+## Setting Up the Blackout Period
+
+1. Go to your target's settings.
+2. In the Scanner tab, locate the **Blackout Period** section.
+3. Enable the feature by toggling the switch component.
+   {{% image src="/deepdive/scans/blackout-period-disabled.png" alt="Blackout Period - Disabled" %}}
+5. Define the start and end time/day in your local timezone for when the scan will be paused and resumed.
+   {{% image src="/deepdive/scans/blackout-period-enabled.png" alt="Blackout Period - Enabled" %}}
+7. Save your changes.
```

Comment:
```suggestion
4. Define the start and end time/day in your local timezone for when the scan will be paused and resumed.
   {{% image src="/deepdive/scans/blackout-period-enabled.png" alt="Blackout Period - Enabled" %}}
5. Save your changes.
```

It renders OK, but let's clean up the markdown. Thanks

<img width="277" alt="Screenshot 2025-01-13 at 19 53 40" src="https://github.com/user-attachments/assets/91fc3685-36f4-4921-98e3-81be322c8841" />